src/System.Reflection.Emit/tests/TypeBuilder/TypeBuilderCreateType.cs

```diff
@@ -29,6 +29,8 @@ public void CreateType(TypeAttributes attributes)
             TypeBuilder type = Helpers.DynamicType(attributes);
             Type createdType = type.CreateTypeInfo().AsType();
             Assert.Equal(type.Name, createdType.Name);
+
+            Assert.Same(type.CreateTypeInfo(), type.CreateTypeInfo());
```

Comment:
Thanks for letting me know. I've fixed this, and all other occurences of `Assert.Same` to `Assert.Equal`